src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+from shipit_static_analysis import Issue
+from cli_common.command import run
+from cli_common.log import get_logger
+import itertools
+import json
+import os
+
+logger = get_logger(__name__)
+
+ISSUE_MARKDOWN = '''
+## mozlint - {linter}
+
+- **Path**: {path}
+- **Level**: {level}
+- **Line**: {line}
+- **In patch**: {in_patch}
+- **Third Party**: {third_party}
+- **Publishable**: {publishable}
+
+```
+{message}
+```
+'''
+
+
+class MozLintIssue(Issue):
+    def __init__(self, repo_dir, repo_path, modified_lines, column, level, lineno, linter, message, **kwargs):
+        self.nb_lines = 1
+        self.repo_dir = repo_dir
+        self.path = repo_path
+        self.column = column
+        self.level = level
+        self.line = lineno
+        self.linter = linter
+        self.message = message
+        self.in_patch = self.line in modified_lines
+
+    def __str__(self):
+        return '{} issue {} {} line {}'.format(
+            self.linter,
+            self.level,
+            self.path,
+            self.line,
+        )
+
+    def is_publishable(self):
+        '''
+        Publishable when:
+        * line is modified by dev.
+        * file is not 3rd party
+        '''
+        return self.in_patch and not self.is_third_party()
+
+    def as_text(self):
+        '''
+        Build the text content for reporters
+        '''
+        return '{}: Linting issue [mozlint: {}]\n\n{}'.format(
```

Comment:
Nit: Please update the format to https://github.com/mozilla-releng/services/issues/757#issuecomment-358309117